.github/workflows/check-clang-format.yml

```
@@ -94,11 +95,27 @@ jobs:
       - name: Install ClangFormat
         run: |
           cd "${{ env.CLANG_FORMAT_INSTALL_PATH }}"
-          tar --extract --file="${{ steps.download.outputs.name }}"
+          tar --extract --file="${{ steps.download-clang-format.outputs.name }}"
           # Add installation to PATH:
           # See: https://docs.github.com/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path
           echo "${{ env.CLANG_FORMAT_INSTALL_PATH }}/clang_Linux_64bit" >> "$GITHUB_PATH"
 
+      - name: Download yq
+        id: download-yq
+        uses: MrOctopus/download-asset-action@1.0
+        with:
+          repository: mikefarah/yq
+          asset: yq_linux_amd64.tar.gz
+          target: ${{ env.YQ_INSTALL_PATH }}
+
+      - name: Install ClangFormat
```

```suggestion
      - name: Install yq
```
leftover from the copy paste

---

Thanks Umberto!